src/extension.ts

```diff
@@ -131,7 +144,7 @@ function activateTOC(
  * @private
  */
 const extension: JupyterFrontEndPlugin<ITableOfContentsRegistry> = {
-  id: 'jupyterlab-toc',
+  id: 'toc',
```

Comment:
```suggestion
  id: '@jupyterlab/toc:plugin',
```

This is similar to the Git extension (see [here](https://github.com/jupyterlab/jupyterlab-git/blob/master/src/index.ts#L40)).

Reply:
Oh I see - do you think this also applies [here](https://github.com/jupyterlab/jupyterlab-toc/blob/e7e73f348a0d5914b35b27c00e24453e4dc0d44d/src/registry.ts#L21)?

Reply:
No, that should be fine, I believe.

Reply:
Oh. Maybe yes. Just to be pedantic. If you can also update that one to also be `@jupyterlab/toc:ITableOfContentsRegistry`, that would be great.